src/docs/product/data-management-settings/scrubbing/advanced-datascrubbing.mdx

```diff
@@ -201,3 +201,5 @@ The following limitations generally apply to all server-side data scrubbing, be
 - Hashing, masking or replacing a JSON object, array or number (anything that is not a string) cannot be done in all circumstances as it would change the JSON type of the value and violate assumptions Sentry's internals make about the data schema. Data scrubbing will ignore the _Method_ in those cases and always remove/replace with `null` as that is always safe.
 
 - Sentry's internals require that the event user's IP address must either be `null` or a valid IPv4/IPv6 address. If you're trying to hash, mask or replace IP addresses, data scrubbing will move the replacement value into the user ID (if one is not already set) in order to avoid breaking this requirement while still providing useful data for the Users count on an issue.
+
+- Scrubbing cannot be done in file names as it would violate assumptions in the processing pipeline resulting in a poor usage experience. You can scrub filenames in the SDK itself.
```

Comment:
I did not mention the integration since there may be better options for their use-case and to not get docs out-of-sync. Happy to include a link if somebody thinks otherwise.

Reply:
I think it does not hurt to add a link to one SDK as an example.

Reply:
Definitely some examples would be nice to have there. Or potential solutions one could use to scrub the data client's side.